### PR TITLE
Improving the about page

### PR DESCRIPTION
**Pull request by mirons-google**

### CODE REVIEW

File: about/index.md [hunk 1]
```diff
@@ -1,48 +1,44 @@
...
+improve the security of your Google Cloud Platform (GCP) environments.
 
 [Get Started]({% link _docs/quickstarts/forseti-security/index.md %}) with
 Forseti Security.
 
-## [Inventory]({% link _docs/quickstarts/inventory/index.md %})
+## Forseti Security
+
+Forseti's core modules come with every deployment of the tool. They can be
+enabled, configured, and executed independtly of each other. When deployed these
```
Comment (carise) [inline about/index.md]: independently

File: about/index.md [hunk 2]
```diff
@@ -1,48 +1,44 @@
...
+
+**[Inventory]({% link _docs/quickstarts/inventory/index.md %})**
 
 Takes a snapshot of resources on a recurring cadence, so you always have a
 history of what was in your cloud.
 
-## [Policy Scanner]({% link _docs/quickstarts/scanner/index.md %})
+**[Policy Scanner]({% link _docs/quickstarts/scanner/index.md %})**
 
 Helps you monitor inventoried GCP resources like Cloud IAM, bigquery,
```
Comment (carise) [inline about/index.md]: BigQuery datasets

**Comment by mirons-google** [comment]
done, thanks!

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as the PR author is a googler at the time of this PR was created.

**Pull request merged**